base/src/main/java/com/smartdevicelink/managers/screen/choiceset/PresentKeyboardOperation.java

```diff
@@ -284,6 +284,8 @@ public void onNotified(RPCNotification notification) {
                 if (onKeyboard.getEvent().equals(KeyboardEvent.ENTRY_VOICE) || onKeyboard.getEvent().equals(KeyboardEvent.ENTRY_SUBMITTED)) {
                     // Submit Voice or Text
                     keyboardListener.onUserDidSubmitInput(onKeyboard.getData(), onKeyboard.getEvent());
+                } else if (onKeyboard.getEvent().equals(KeyboardEvent.INPUT_KEY_MASK_ENABLED) || onKeyboard.getEvent().equals(KeyboardEvent.INPUT_KEY_MASK_DISABLED)) {
```

Comment:
@kostyaBoss Since these enums were most recently added I would move the `else if block` on lines 287-288 below the last else if (around line 310).